src/vivarium/framework/lookup/table.py

```diff
@@ -173,11 +183,19 @@ def call(self, index: pd.Index) -> pd.DataFrame:
         """
         pop = self.population_view.get(index)
         del pop["tracked"]
+        if self.clock is None:
```

Comment:
If `clock` is defined as an attribute of `InterpolatedTable`, it can be type-hinted as not being `Optional`, so this check would not be necessary.